tasks/manage_app.yml

```diff
@@ -20,37 +26,33 @@
     - route
 
 - name: Display OpenShift's deployments for this app
-  debug:
-    var=deployments
+  debug: msg="deployments={{ deployments | to_nice_yaml}}"
   tags:
     - deploy
-    - deployment
+    - deployments
```

Comment:
What is the purpose of having `deploy` and `deployments` tags?

Reply:
Sorry it's your code : https://github.com/openfun/arnold/blob/master/tasks/deploy_app.yml
But If whe want we can filter the deploy only on deployment or service or whatever.

Reply:
I'm agree :+1: